dired-collapse.el

```diff
@@ -80,6 +80,7 @@
   (if dired-collapse-mode
       (progn
         (add-hook 'dired-after-readin-hook 'dired-collapse 'append 'local)
+        (add-hook 'dired-subtree-after-insert-hook 'dired-collapse 'append 'local)
```

Comment:
We should also remove the hook a couple lines below when we disable the mode.

Reply:
Whoops, good catch, will do.

Reply:
done.